src/core/data.jl

```diff
@@ -2199,16 +2251,17 @@ end
 
 
 """
-propages inactive active network buses status to attached components so that
+propagates inactive active network buses status to attached components so that
 the system status values are consistent.
 
 returns true if any component was modified.
 """
-function propagate_topology_status!(data::Dict{String,<:Any})
+function propagate_topology_status!(data_all::Dict{String, <:Any})
```

Comment:
I would suggest `data` remain the naming convention for what is called `data_all` now.  inside a function's scope I would change `data` to `pm_data` and `nw_data` to `pm_nw_data`.

Reply:
Sounds good, will change this.